<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>noraMDagNodeSelectWidget</class>
 <widget class="QDialog" name="noraMDagNodeSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="selectLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>label</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
